sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/CollationTypeCasts.scala

```diff
@@ -18,11 +18,9 @@
 package org.apache.spark.sql.catalyst.analysis
 
 import javax.annotation.Nullable
-
```

Comment:
do we need this change? seems unrelated + afaik these imports should be separated by an empty line